utilities/cli/holohub.py

```diff
@@ -393,6 +394,7 @@ def handle_run(self, args: argparse.Namespace) -> None:
                     "Ninja",
                     f"-DCMAKE_BUILD_TYPE={BUILD_TYPE}",
                     f"-DCMAKE_PREFIX_PATH={HoloHubCLI.DEFAULT_SDK_DIR}",
+                    f"-DHOLOHUB_DATA_DIR:PATH={HoloHubCLI.DEFAULT_DATA_DIR}",
```

Comment:
out of scope, but can't we leverage/call whats in `handle_build` from `handle_run` ?

Reply:
Yep, we should refactor this as part of @wyli 's suggestions to make CLI tooling more accessible for gen AI calls. Deferring for now.